src/faq.md

```diff
@@ -0,0 +1,55 @@
+# FAQ
+
+- [Doesn't TLS allow a third party to verify data authenticity?](#faq1)
+- [How exactly does a Verifier participate in the TLS connection?](#faq2)
+- [What are the trust assumptions of the TLSNotary protocol?](#faq3)
+- [What is the role of a Notary?](#faq4)
+- [Is the Notary an essential part of the TLSNotary protocol?](#faq5)
+- [Which TLS versions are supported?](#faq6)
+- [What is the overhead of using the TLSNotary protocol?](#faq7)
+
+
+### Doesn't TLS allow a third party to verify data authenticity? { #faq1 }
+
+No, it does not. TLS is designed to guarantee the authenticity of data **only to the participants** of the TLS connection. TLS does not have a mechanism to enable the server to "sign" the data.
+
+The TLSNotary protocol overcomes this limitation by making the third-party `Verifier` a participant in the TLS connection. 
+
+### How exactly does a Verifier participate in the TLS connection? { #faq2 }
+
+The `Verifier` collaborates with the `Prover` using secure multi-party computation (MPC). There is no requirement for the `Verifier` to monitor or to access the `Prover's` TLS connection. The `Prover` is the one who communicates with the server.
+
+### What are the trust assumptions of the TLSNotary protocol? { #faq3 }
+
+The protocol does not have trust assumptions. In particular, it does not rely on secure hardware or on untamperability of the communication channel.
+
+The protocol does not rely on participants to act honestly. Specifically, it guarantees that, on the one hand, a malicious `Prover` will not be able to convince the `Verifier` of the authenticity of false data, and, on the other hand, that a malicious `Verifier` will not be able to learn the private data of the `Prover`.
+
+### What is the role of a Notary? { #faq4 }
+
+In some scenarios where the `Verifier` is unable to participate in a TLS connection, they may choose to delegate the verification of the online phase of the protocol to an entity called the `Notary`.
+
+Just like the `Verifier` would ([see FAQ above](#faq2)), the `Notary` collaborates with the `Prover` using MPC to enable the `Prover` to communicate with the server. At the end of the online phase, the `Notary` produces an attestation trusted by the `Verifier`. Then, in the offline phase, the `Verifier` is able to ascertain data authenticity based on the attestation.
+
+### Is the Notary an essential part of the TLSNotary protocol? { #faq5 }
+
+No, it is not essential. The `Notary` is an optional role which we introduced in the `tlsn` library as a convenience mode to support `Verifiers` who may choose not to participate in the TLS connection themselves.
```

Comment:
```suggestion
No, it is not essential. The `Notary` is an optional role which we introduced in the `tlsn` library as a convenience mode for `Verifiers` who choose to outsource the MPC-TLS verification to a `Notary`.
```

Reply:
I was deliberately trying to avoid the term `MPC-TLS verification`. I didnt want to prime the reader to the idea that there is MPC-TLS verification and app-specific data verification.
I just wanted a simple takeaway: "the Verifier cant be online, so the Verifier needs a Notary".
Could you re-evaluate you suggestion in this light and tell me if you agree?

Reply:
Fine for me!